requirements.txt

```diff
@@ -1 +1,2 @@
-ops >= 2.2.0
+ops ~= 2.17
```

Comment:
Can you pin this deps to exact version so that we are closer to repeatable builds? We can rely in renovate to open a PR for every dependency update